src/elements/image/image.visual.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { html } from 'lit/static-html.js';
+
+import sampleImages from '../core/images.js';
+import { describeEach, describeViewports, visualDiffDefault } from '../core/testing/private.js';
+import { waitForImageReady } from '../core/testing.js';
+
+import './image.js';
+
+const imageUrl = import.meta.resolve('../core/testing/assets/lucerne.png');
+
+describe(`sbb-image`, () => {
+  const aspectRatioCases = {
+    aspectRatio: [
+      'free',
+      '1-1',
+      '1-1',
+      '1-2',
+      '2-1',
+      '2-3',
+      '3-2',
+      '3-4',
+      '4-3',
+      '4-5',
+      '5-4',
+      '9-16',
+      '16-9',
+    ],
+  };
+
+  const additionalBorderRadiusCases = {
+    borderRadius: ['none', 'round'],
+  };
+
+  describeViewports({ viewports: ['zero', 'medium'] }, () => {
+    describeEach(aspectRatioCases, ({ aspectRatio }) => {
```

Comment:
Nitpick; I'm not sure whether we should use the describeEach for single parameters. We could use just a for loop here with the it here. Thoughts?